src/sonic-config-engine/sonic-cfggen

```diff
@@ -137,6 +138,31 @@ def ip_network(value):
         return "Invalid ip address %s" % value
     return r_v.network
 
+
+def b64encode(value):
```

Comment:
Where will b64encode get called? Could you please provide an example?

Reply:
see `files/image_config/ntp/ntp.keys.j2` file from that PR. it used there